src/js/views/Project/Configuration/ViewSSMParam.jsx

```diff
@@ -1,18 +1,71 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { DisplaySSMParam } from './DisplaySSMParam'
-import { DisplaySecureStringParam } from './DisplaySecureStringParam'
+import { useTranslation } from 'react-i18next'
+import { DescriptionList } from '../../../components/DescriptionList/DescriptionList'
+import { Definition } from '../../../components/DescriptionList/Definition'
+import { Toggle } from '../../../components/Form/Toggle'
+import { DefinitionRow } from '../../../components/DescriptionList/DefinitionRow'
 
-function ViewSSMParam({ param }) {
-  if (param.type === 'SecureString') {
-    return <DisplaySecureStringParam param={param} />
-  } else {
-    return <DisplaySSMParam param={param} />
-  }
+function ViewSSMParam({ param, showSecureStrings, onShowSecureStringsChange }) {
+  const { t } = useTranslation()
+
+  const allTypes = new Set(param.values.map((value) => value.type))
+  const includesSecureString = allTypes.has('SecureString')
+  const includesMultipleTypes = allTypes.size > 1
+  const typeDisplay = includesMultipleTypes
+    ? Array.from(allTypes).join(', ')
+    : param.values[0].type
```

Comment:
I think that this could be calculated in `SSMConfiguration` where you call `setRows()` unless I am missing something. `param.type` should be the comma-separated value already. You should be able to pre-calc `includesMultipleTypes` as well as `has('SecureString')`.

Reply:
Ah you're right, I did this first then moved it out to `SSMConfiguration` and then didn't update it here to use it.